java/javax/servlet/http/Part.java

```diff
@@ -48,9 +48,6 @@
     public String getContentType();
 
     /**
-     * Obtain the name of the field in the multipart form corresponding to this
-     * part.
-     *
      * @return The name of the field in the multipart form corresponding to this
```

Comment:
-1 to this and all similar changes. The first sentence in the Javadoc is used as the method description in the table of methods. By removing this text, the description is lost from that table. While I am never a fan of duplication, in this case it is better to have the duplication than not.